great_expectations/data_context/store/store.py

```diff
@@ -165,9 +165,15 @@ def set(self, key: DataContextKey, value: Any, **kwargs: dict) -> None:
             return self._store_backend.set(key, value, **kwargs)
 
         self._validate_key(key)
-        return self._store_backend.set(
-            self.key_to_tuple(key), self.serialize(value), **kwargs
-        )
+        # TODO: AJB 20220803 remove this isinstance call
+        if isinstance(self._store_backend, GeCloudStoreBackend):
+            return self.ge_cloud_response_json_to_object_dict(
+                response_json=self._store_backend.set(key, value, **kwargs)
+            )
```

Comment:
Temporary sidestep of conversion to tuple for GeCloudStoreBackend until we can make all store backends use real keys and configs.